buildIncremental.fsx

```diff
@@ -42,18 +42,37 @@ module IncrementalTests =
         | EndsWith "cmd" -> true
         | EndsWith "sh" -> true
         | _ -> false
+    
+    let getHeadHashFor repositoryDir branch =
+        let _, msg, error = runGitCommand repositoryDir (sprintf "log --oneline -1 %s" branch)
+        if error <> "" then failwithf "git log --oneline failed: %s" error
+        let logMsg = msg |> Seq.head
+        let tmp =
+            logMsg.Split(' ')
+            |> Seq.head
+            |> fun s -> s.Split('m')
+        if tmp |> Array.length > 2 then tmp.[1].Substring(0,6) else tmp.[0].Substring(0,6)
 
     let getUpdatedFiles() = 
         let srcDir = __SOURCE_DIRECTORY__
         let localBranches = getLocalBranches srcDir
+        log "Local branches..."
+        localBranches |> Seq.iter log
         if not (localBranches |> Seq.exists (fun b -> b = "v1.3")) then
+            log "v1.3 branch information not available... fetching"
```

Comment:
Extract the v1.3 value into a constant somewhere named defaultbranch or something. That way its more explicit what it means, and easier to change once we move to a different default branch.